frontend/pages/Homepage/_styles.scss

```diff
@@ -51,20 +47,36 @@
     }
   }
 
+  &__platforms {
+    display: flex;
+    align-items: center;
+
+    span {
+      font-size: $small;
+    }
+    .Select-control, .Select-value {
+      background-color: $core-white !important;
+    }
+  }
+
+  &__platform_dropdown {
+    width: 138px;
+  }
+
   &__section {
     margin-top: $pad-medium;
     width: 100%;
     display: grid;
+    row-gap: $pad-medium;
 
-    &.two-column {
-      row-gap: $pad-medium;
+    // if we have an odd number of elements, take 
+    // the last one and stretch it across the grid
+    .homepage-info-card:nth-child(odd):last-child {
+      grid-column-start: span 2;
```

Comment:
Great idea! 💡

Reply:
I felt very accomplished figuring that out lol. Thanks for noticing!